libraries/AP_NavEKF3/AP_NavEKF3_Control.cpp

```diff
@@ -611,9 +611,12 @@ bool NavEKF3_core::setOrigin(const Location &loc)
     validOrigin = true;
     GCS_SEND_TEXT(MAV_SEVERITY_INFO, "EKF3 IMU%u origin set",(unsigned)imu_index);
 
-    // put origin in frontend as well to ensure it stays in sync between lanes
-    frontend->common_EKF_origin = EKF_origin;
-    frontend->common_origin_valid = true;
+    if (!frontend->common_origin_valid) {
```

Comment:
This means that the common shared origin for the reported NED positions can only ever be set once which is OK for this PR given the vehicle code currently does not require to be able to change origin. @rmackay9 FYI